config/local/config.yml

```diff
@@ -19,238 +19,198 @@ fund_rebalancer:
-  arbitrum-testnet:
-    chain_name: "arbitrum"
-    chain_id: "421614"
-    type: evm
-    environment: "testnet"
-    gas_token_symbol: "ETH"
-    gas_token_decimals: 18
-    evm:
-      rpc: <arbitrum_rpc_server_url>
-      rpc_basic_auth_var: <server_password>
-      contracts:
-        usdc_erc20_address: 0x75faf114eafb1BDbe2F0316DF893fd58CE46AA4d
-      signer_gas_balance:
-        warning_threshold_wei: 250000000000000000
-        critical_threshold_wei: 0
-  osmosis:
-    chain_name: "osmosis"
-    chain_id: "osmosis-1"
-    type: "cosmos"
-    environment: "mainnet"
-    gas_token_symbol: "OSMO"
-    gas_token_decimals: 6
-    fast_transfer_contract_address: <use_address_from_readme>
-    solver_address: <solver_address>
-    hyperlane_domain: "875"
-    num_block_confirmations_before_fill: <num_block_confirmations_before_fill> # e.g. 1
-    batch_uusdc_settle_up_threshold: <batch_uusdc_settle_up_threshold> # e.g. 5000000
-    min_fill_size: <min_fill_size> # e.g. 5000000
-    max_fill_size: <max_fill_size> # e.g. 10000000000
-    max_rebalancing_gas_threshold: <max_rebalancing_gas_threshold> # e.g. 1000000
-    min_fee_bps: <min_fee_bps> # e.g. 100
-    usdc_denom: "ibc/498A0751C798A0D9A389AA3691123DADA57DAA4FE165D5C75894505B876BA6E4"
-    cosmos:
-      address_prefix: "osmo"
-      rpc: <osmosis_rpc_server_url> # e.g. "https://rpc.osmosis.zone"
-      rpc_basic_auth_var: <server_password>
-      grpc: <osmosis_grpc_server_url> # e.g. "grpc.osmosis.zone:9090"
-      grpc_tls_enabled: <grpc_tls_enabled> # e.g. false
-      usdc_denom: "ibc/498A0751C798A0D9A389AA3691123DADA57DAA4FE165D5C75894505B876BA6E4"
-      signer_gas_balance:
-        warning_threshold_wei: <warning_threshold_wei> # e.g. 42860000
-        critical_threshold_wei: <critical_threshold_wei> # e.g. 14290000
-      gas_price: <gas_price> # e.g. "0.0025"
-      gas_denom: "uosmo"
-    relayer:
-      validator_announce_contract_address: "osmo147r8mfdsngswujgkr4tln9rhcrzz6yq0xn448ksd96mlcmp9wg6stvznke"
-      merkle_hook_contract_address: "osmo1e765uc5mctl7rz8dzl9decl5ghgxggeqyxutkjp2xkggrg6zma3qgdq2g4"
-      mailbox_address: "osmo1r6u37zv47ke4d2k9tkzun72ch466w6594kv8gqgrtmsvf7qxpm9sj95v98"
 
-  base-mainnet:
+  8453:
     chain_name: "base"
     chain_id: "8453"
-    type: evm
+    type: "evm"
     environment: "mainnet"
     gas_token_symbol: "ETH"
     gas_token_decimals: 18
-    fast_transfer_contract_address: <use_address_from_readme>
     num_block_confirmations_before_fill: <num_block_confirmations_before_fill> # e.g. 1
-    batch_uusdc_settle_up_threshold: <batch_uusdc_settle_up_threshold> # e.g. 5000000
-    min_fill_size: <min_fill_size> # e.g. 5000000
-    max_fill_size: <max_fill_size> # e.g. 10000000000
+    hyperlane_domain: "8453"
+    quick_start_num_blocks_back: <quick_start_num_blocks_back> # e.g. 1000
     max_rebalancing_gas_threshold: <max_rebalancing_gas_threshold> # e.g. 1000000
-    min_fee_bps: <min_fee_bps> # e.g. 100
```

Comment:
did this get removed by mistake? people should be setting the min fee bps for evm chains